### ISSUE TITLE
docker env: ALLOWED_HOSTS issue

### ISSUE THREAD

**Issue by mhow2** [Bitergia/prosoul#86]
I tried the PR #85 like follow. 
- updated the docker image with `docker pull acsdocker/prosoul`
- Edited by docker-compose by adding my new `host.domain.ltd` to the variable:
```yaml
    prosoul:
        image: acsdocker/prosoul
        ports:
         - "8000:8000"
        depends_on:
         - kibiter
         - elasticsearch
        environment:
          - ALLOWED_HOSTS="prosoul localhost 172.17.0.1 127.0.0.1 host.domain.ltd"
```
- then I go to `host.domain.ltd:8000` and I get the following error:
```
Request Method: | GET
-- | --
http://host.domain.ltd:8000/
2.1.3
DisallowedHost
Invalid HTTP_HOST header: 'host.domain.ltd:8000'. You may need to add 'host.domain.ltd' to ALLOWED_HOSTS.
/usr/local/lib/python3.5/dist-packages/django/http/request.py in get_host, line 106
/usr/bin/python3
3.5.3
['/home/prosoul/prosoul/django-prosoul',  '/usr/lib/python35.zip',  '/usr/lib/python3.5',  '/usr/lib/python3.5/plat-x86_64-linux-gnu',  '/usr/lib/python3.5/lib-dynload',  '/usr/local/lib/python3.5/dist-packages',  '/usr/lib/python3/dist-packages']
Mon, 26 Nov 2018 13:34:31 +0000
```

* I Also verified the ALLOWED_HOSTS variable within the container : 

```
$ docker exec 5e89e8d59cf4 env |grep ALLOWED_HOST
ALLOWED_HOSTS="prosoul localhost 172.17.0.1 127.0.0.1 host.domain.ltd"
```

**Comment by valeriocos** [comment]
Fixed with #88 . @mhow2  feel free to close the issue if you don't detect any problem.

**Comment by mhow2** [comment]
Thanks to the fix, I've been able to run the software properly.
I first hit Error 400, then after having removed the double quote around the `ALLOWED_HOSTS` variable values it worked.
